### PR TITLE
Create: 3 IOKs for common Steam phishing kits

### PR DESCRIPTION
**Pull request by PCPisChill**
Adds IOKs for common Steam phishing kits that get spammed in Steam groups:

Steam CSGO2 Beta Phishing Kit

- https://urlscan.io/result/de077e20-ab89-494b-af4c-df49f72d1e8b
- https://urlscan.io/result/2fca4b90-38da-4880-9b09-14e3a94c68e6
- https://urlscan.io/result/558a4fb2-8e16-4176-9a16-ebfe2fe64d2b
- https://urlscan.io/result/0712a363-be77-4482-960a-886738d7f882
- https://urlscan.io/result/950664a4-aaf8-4d2d-8761-c0b2fa11ab7a
- https://urlscan.io/result/1daf0866-8168-4efe-9f37-067b89b886b4
- https://urlscan.io/result/6f719b9d-0ded-4f3c-ba3f-682b1afd015f
- https://urlscan.io/result/bd1cef80-59d3-43cb-bfb3-b71a9cc411b0
- https://urlscan.io/result/5c36ed3f-3efe-43a9-a669-f13f4ff0cdcb
- https://urlscan.io/result/ed5b5e51-109f-4dec-b886-6a49b44e8fe8

Steam Phishing Kit getsiteconfig:

- https://urlscan.io/result/732d40f3-c113-44da-bcd4-5f39ff173e83
- https://urlscan.io/result/0712a363-be77-4482-960a-886738d7f882
- https://urlscan.io/result/01e4685b-9001-4843-a50f-a41ad126fc8c
- https://urlscan.io/result/64c8c423-5e1e-4779-a4b0-66c9e0beb8d7
- https://urlscan.io/result/02d78cc5-5035-490d-ade3-8043a1d29d29
- https://urlscan.io/result/65902fde-168e-4492-a039-b678cedc23c8
- https://urlscan.io/result/5988575a-6cc7-4e5b-ac87-22fb5e29b6e6
- https://urlscan.io/result/0da6a52c-b3bd-4044-ab1f-dc0efbaee16b

Steam Auronplay Gift Card Phishing Kit:

- https://urlscan.io/result/732d40f3-c113-44da-bcd4-5f39ff173e83
- https://urlscan.io/result/ee34fa99-6cf8-4b16-8cf5-e617e238dea0
- https://urlscan.io/result/d09c1f36-773f-437c-b533-4dced6cecc1f
- https://urlscan.io/result/23b2c035-4daa-405e-98cd-0f3cdddcd5ca
- https://urlscan.io/result/0b26dcba-e480-4690-a856-f8b3577194ec
- https://urlscan.io/result/5d409d0e-4f1d-4a8f-b076-b473c9398a43

### CODE REVIEW

File: indicators/steam-getsiteconfig.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  Steam Phishing Kit with obfuscated javascript pages that use a fake Steam login window to steal user credentials and free 50/100$ gift cards, csgo skins, csgo2 beta or discord nitro as bait.
+
+references:
+  - https://urlscan.io/result/732d40f3-c113-44da-bcd4-5f39ff173e83
+  - https://urlscan.io/result/0712a363-be77-4482-960a-886738d7f882
+  - https://urlscan.io/result/01e4685b-9001-4843-a50f-a41ad126fc8c
+  - https://urlscan.io/result/64c8c423-5e1e-4779-a4b0-66c9e0beb8d7
+  - https://urlscan.io/result/02d78cc5-5035-490d-ade3-8043a1d29d29
+  - https://urlscan.io/result/65902fde-168e-4492-a039-b678cedc23c8
+  - https://urlscan.io/result/5988575a-6cc7-4e5b-ac87-22fb5e29b6e6
```
Comment (IlluminatiFish) [inline indicators/steam-getsiteconfig.yml:13]: It appears that this reference already matches an existing rule (`steam-4540135`)

File: indicators/steam-metrica.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - https://urlscan.io/result/d7243f7d-e10a-4309-8450-869229ac286d
+  - https://urlscan.io/result/1ec85742-f31e-4d45-a37b-8c91d94996df
+  - https://urlscan.io/result/d09c1f36-773f-437c-b533-4dced6cecc1f
+  - https://urlscan.io/result/0eb7900f-dd6e-4491-ad69-e9f4b14e0e41
+  - https://urlscan.io/result/19b11718-4018-449f-8829-3365a10c812a
+  - https://urlscan.io/result/8690e751-6bb9-4bd7-a7b9-34ce3b719fef
+
+detection:
+
+  getSiteConfigJson:
```
Comment (IlluminatiFish) [inline indicators/steam-metrica.yml:16]: I think there's been a mistake here with the field name

**Comment by IlluminatiFish** [comment]
Please resolve the issues identified by the workflow

**Comment by PCPisChill** [comment]
> Please resolve the issues identified by the workflow

Should all be working now, not sure why 'steam-auronplay.yml' wasn't working for https://urlscan.io/result/23b2c035-4daa-405e-98cd-0f3cdddcd5ca as 'giftFromAuronplay' should have caught it but I've updated it to regex.

**Review by IlluminatiFish — CHANGES_REQUESTED** [review]
The reference scan you mention in your change does not appear to match either the original detection logic nor the updated logic with the regular expression. I believe this reference should be removed from this rule's reference list.

**Review by IlluminatiFish — APPROVED** [review]

**Pull request merged**